packet/gtp.go

```diff
@@ -0,0 +1,138 @@
+// Copyright 2017 Intel Corporation.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+// This file highly assumes that all packets in each particular situation
+// will have the same structure. Parsing, Encapsulating, Decapsulating
+// functions are base variants without any checkings which can be
+// painlessly changed for exact situation.
+
+package packet
+
+import (
+	"fmt"
+	"unsafe"
+
+	. "github.com/intel-go/yanff/common"
```

Comment:
Usage of dot imports outside of testing code is discouraged:
See https://github.com/golang/go/wiki/CodeReviewComments#import-dot.

Reply:
Many go static code analyzers can't handle dot-imports.
Also, `go lint` will say: `should not use dot imports`.